test/test_onnx.py

```diff
@@ -0,0 +1,73 @@
+# Some standard imports
+import numpy as np
+import io
+import torch
+from torch import nn
+from torch.autograd import Variable
+import torch.nn.init as init
+import unittest
+import tempfile
+import shutil
+
+
+class SuperResolutionNet(nn.Module):
```

Comment:
For a smoketest isn't it better to pick a model that runs quickly, without using lots of memory?

Reply:
Also, it is probably easier if you put the tests in `test_jit.py`, otherwise you will have to update the `run_tests.py` scripts etc to pick up this new file. And it will also complicate things when the onnx-fb-universe tests come back in because we will have to merge the files.